Mathlib/Algebra/GroupPower/Lemmas.lean

```diff
@@ -630,9 +631,9 @@ theorem neg_one_pow_eq_pow_mod_two [Ring R] {n : ℕ} : (-1 : R) ^ n = (-1) ^ (n
   rw [← Nat.mod_add_div n 2, pow_add, pow_mul]; simp [sq]
 #align neg_one_pow_eq_pow_mod_two neg_one_pow_eq_pow_mod_two
 
-section StrictOrderedSemiring
+section OrderedSemiring
 
-variable [StrictOrderedSemiring R] {a : R}
+variable [OrderedSemiring R] {a : R}
```

Comment:
I'm a little wary of this one: didn't we decide `OrderedSemiring` was nonsense? Maybe @YaelDillies remembers the history better.

Reply:
I believe Yaël fixed the weirdness a while ago: [mathlib#16172](https://github.com/leanprover-community/mathlib/pull/16172).

Reply:
By the way, Yaël is on vacation this week, according to Zulip. So I don't want to block on them :)

Reply:
Just came back from my holidays. Anne is perfectly right!